nestkernel/archiving_node.cpp

```diff
@@ -66,6 +68,8 @@ nest::Archiving_Node::Archiving_Node( const Archiving_Node& n )
   , beta_Ca_( n.beta_Ca_ )
   , synaptic_elements_map_( n.synaptic_elements_map_ )
 {
+  tau_minus_inv_ = 1. / tau_minus_;
```

Comment:
This should be copied directly from `const Archiving_Node& n` rather than recomputed from the base value.